src/inverted-index.js

```diff
@@ -1,85 +1,89 @@
-'use strict';
-class Index {
 /**
- * Get Index method that indexes JSON file
- * @param data (key,value pair)
- * @return fileIndex
+ * Function that regularizes input
+ * @param {String} tokens
+ * @return {Array} newToken
  */
-    getIndex(data) {
-        let fileIndex = {};
-        for(let i=0;i<data.docs.length;i++) {
-            fileIndex[data.name] = [i];
-        }
-        return fileIndex;
+const tokenize = (tokens) => {
```

Comment:
I believe you're trying to declare a class function here.

Firstly, it shouldn't be outside the class.

Secondly, why are you declaring functions with the keyword `const`?